src/meson.build

```diff
@@ -0,0 +1,17 @@
+raylib_src = [
+  'raudio.c',
+  'rcore.c',
+  'rglfw.c',
```

Comment:
rglfw seems to be an amalgamation of the glfw source code, it is redundant both:
- when using an external dependency on glfw
- when using a vendored copy (subproject wrap, or src/external/glfw/ checked into git)